modules/dialogic_consoleio/__init__.py

```diff
@@ -5,20 +5,20 @@
 
 
 @state(read="rawio:in")
-def console_shutdown(sess):
-    rawin = sess["rawio:in"]
+def console_shutdown(ctx):
+    rawin = ctx["rawio:in"]
```

Comment:
I'd export "rawio:in" as a parameter because later it might be hard to change the name